packages/router/src/directives/router_link_active.ts

```diff
@@ -99,6 +108,7 @@ export class RouterLinkActive implements OnChanges, OnDestroy, AfterContentInit
    */
   @Input() routerLinkActiveOptions: {exact: boolean}|IsActiveMatchOptions = {exact: false};
 
+  @Output() public readonly routerLinkActivated: EventEmitter<boolean> = new EventEmitter();
```

Comment:
```suggestion
  @Output() public readonly isActiveChange: EventEmitter<boolean> = new EventEmitter();
```
Bike-shedding: I'm not sure how much I like "routerLinkActivated". How about `isActiveChange`?